libs/lib_parsing/Parsed_int.mli

```diff
@@ -0,0 +1,43 @@
+(* Why concrete integers?
+ * This is a library for "concrete ints", which are a type that correspond to
+ * literal integers that have been parsed by the Semgrep engine.
```

Comment:
try not to mention Semgrep here. Code in libs/ is supposed to be independent of Semgrep.
You can use Semgrep as an example, but better avoid if possible, to make the code reusable
in other context possibly.